packages/components/src/components/segmented-control/segmented-control/segmented-control-utils.ts

```diff
@@ -17,6 +17,9 @@ export type SegmentedControlBackgroundColor = (typeof SEGMENTED_CONTROL_BACKGROU
 
 export type SegmentedControlUpdateEvent = { value: string | number };
 
+export const SEGMENTED_CONTROL_COLUMNS = [...Array.from({ length: 25 }, (_, i) => i + 1), 'auto'];
```

Comment:
If `'auto'` was be the first value we wouldn't need to iterate 25 times for the default use case on each prop change.